ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -72,66 +86,63 @@ module.exports = Marionette.LayoutView.extend({
   handleTarget: function() {
     this.$el.toggleClass('has-feature', this.model.get('target') !== undefined)
   },
-  serializeData: function() {
-    let modelJSON = this.model.toJSON()
-    let summaryModel = {}
-    const summary = properties.summaryShow
-
-    summary.forEach(attribute => {
-      if (this.model.get('targetMetacard') !== undefined) {
-        const attributeName = hbHelper.getAlias(attribute)
-        const attributeValue = this.model
-          .get('targetMetacard')
-          .get('metacard')
-          .get('properties')
-          .get(attributeName)
-        if (attributeValue !== undefined)
-          summaryModel[attributeName] = massageData(
-            attributeName,
-            attributeValue
-          )
-      }
-    })
-
-    let viewData = {
-      summary: summaryModel,
-      target: modelJSON.target,
-      lat: modelJSON.mouseLat,
-      lon: modelJSON.mouseLon,
+  getDisplayComponent: function() {
+    const coordinateFormat = getCoordinateFormat()
```

Comment:
✏️ 	Can we move this to right above the `switch` statement? It's not used in the next 4 lines